### ISSUE TITLE
Seperate training on MotionNet

### ISSUE THREAD

**Issue by DonaldKam** [bryanyzhu/Hidden-Two-Stream#44]
That was great job. Can the **MotionNet** be separately trained with ground truth optical flow images？

**Comment by bryanyzhu** [comment]
Yes, you can. Instead of using photometric loss at [here](https://github.com/bryanyzhu/Hidden-Two-Stream/blob/master/models/multiframe_MotionNet/train.prototxt#L1333-L1347), you can load GT optical flow images and compute L1 loss.
